### PR TITLE
Bug fix property and methods of BackendV2Converter

### PR DESCRIPTION
**Pull request by to24toro**
### Summary
I found and fixed bugs of `BackendV2Converter`.

### Details and comments
Important and critical bugs to use BackendV2Converter.

### CODE REVIEW

**Review by nkanazawa1989 — COMMENTED** [review]
Thanks @to24toro , can you write release note?

File: releasenotes/notes/fix-backendv2converter-de342352cf882494.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Fixed a bug where :meth:`BackendV2Converter.meas_map` did not return correct value.
```
Comment (nkanazawa1989) [inline releasenotes/notes/fix-backendv2converter-de342352cf882494.yaml:4]:
```suggestion
    Fixed a bug of wrong return value of the method :meth:`BackendV2Converter.meas_map` 
    that had returned backend `dt` instead.
```

File: releasenotes/notes/fix-backendv2converter-de342352cf882494.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    Fixed a bug where :meth:`BackendV2Converter.drive_channel`, 
+    :meth:`BackendV2Converter.measure_channel`, :meth:`BackendV2Converter.acquire_channel` and 
+    :meth:`BackendV2Converter.control_channel` did not return any value.
```
Comment (nkanazawa1989) [inline releasenotes/notes/fix-backendv2converter-de342352cf882494.yaml:7]:
```suggestion
  - |
    Fixed a bug of missing return value from methods :meth:`BackendV2Converter.drive_channel`, 
    :meth:`BackendV2Converter.measure_channel`, :meth:`BackendV2Converter.acquire_channel` and 
    :meth:`BackendV2Converter.control_channel`.

```

**Review by nkanazawa1989 — COMMENTED** [review]
Thanks. LGTM. Just a minor rephrasing before merge.

File: qiskit/providers/backend_compat.py [hunk 3]
```diff
@@ -266,19 +266,19 @@ def dtm(self) -> float:
 
     @property
     def meas_map(self) -> List[List[int]]:
-        return self._config.dt
+        return self._config.meas_map
 
     def drive_channel(self, qubit: int):
-        self._config.drive(qubit)
+        return self._config.drive(qubit)
```
Comment (mtreinish) [inline qiskit/providers/backend_compat.py:272]: Lol, this is my bad. I switch back and forth between rust and python too much (Rust has an implicit return on the last line of a function without a semicolon)

**Review by mtreinish — APPROVED** [review]
LGTM, thanks for catching and fixing this.

File: qiskit/providers/backend_compat.py (hunk 3 above)
Comment (to24toro) [reply qiskit/providers/backend_compat.py:272]: Certainly, the reason is very persuasive!

**Pull request merged**